hw/top_earlgrey/doc/design/README.md

```diff
@@ -379,7 +379,7 @@ The choice of memory, or lack thereof at location 0x0 confers two exclusive bene
 
 For the purpose of `top_earlgrey`, the first option has been chosen to benefit software development and testing
 
-<!-- BEGIN DOCGEN util/design/gen-top-docs.py -t hw/top_earlgrey/data/autogen/top_earlgrey.gen.hjson -g mmap -->
+<!-- BEGIN CMDGEN util/design/gen-top-docs.py -t hw/top_earlgrey/data/autogen/top_earlgrey.gen.hjson -g mmap -->
```

Comment:
Why do you prefer the name command generator as opposed to documentation generator?

Reply:
The name change from a discussion about DOCGEN being quite generic and not very descriptive. We have a lot of document generators: `rustdoc`, `doxygne`, `selfdoc`, etc.

I feel `CMDGEN` better describes what the tool does; It simply runs a command to generate content.

Reply:
Ok, since @jwnrt seems to be happy with this change, I think we can keep it.